site/src/pages/TemplatePage/TemplateSummaryPage/DAUChart.tsx

```diff
@@ -71,7 +71,7 @@ export const DAUChart: FC<DAUChartProps> = ({
   defaults.font.family = theme.typography.fontFamily as string
   defaults.color = theme.palette.text.secondary
 
-  const options = {
+  const options: ChartOptions<'line'> = {
```

Comment:
[Suggested by docs](https://react-chartjs-2.js.org/faq/typescript/) to infer the correct `options`